extensions/dashlane-vault/src/master-password.tsx

```diff
@@ -0,0 +1,66 @@
+import { Action, ActionPanel, Form, LocalStorage, Toast, showToast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { getMasterPassword } from "./helper/master-password";
+import { setKeychainItem } from "./lib/authcli";
+import { encryptVault } from "./lib/dcli";
+
+export default function SetMasterPassword() {
+  const [password, setPasswordInput] = useState("");
+  const [keyName, setKeyName] = useState("");
+  const [passwordExists, setPasswordExists] = useState(false);
+
+  const fetchKeyNameAndCheckPassword = async () => {
+    const existingPassword = await getMasterPassword();
+    if (existingPassword) {
+      setPasswordExists(true);
+      setPasswordInput(existingPassword);
+    } else {
+      const keyName = `dashlane-vault-master-password-${Math.random().toString(36).substring(2, 15)}`;
```

Comment:
Is there the need of the random key name? I thought that the keychain has a "space" for each application. If so we just could use a static `keyName`.